drivers/syslog/vsyslog.c

```diff
@@ -201,17 +202,26 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 #endif
 
 #if defined(CONFIG_SYSLOG_PRIORITY)
-  /* Pre-pend the message priority. */
+  /* Prepend the message priority. */
 
   ret += lib_sprintf(&stream.public, "[%6s] ", g_priority_str[priority]);
 #endif
 
 #if defined(CONFIG_SYSLOG_PREFIX)
-  /* Pre-pend the prefix, if available */
+  /* Prepend the prefix, if available */
 
   ret += lib_sprintf(&stream.public, "%s", CONFIG_SYSLOG_PREFIX_STRING);
 #endif
 
+#if CONFIG_TASK_NAME_SIZE > 0 && defined(CONFIG_SYSLOG_PROCESS_NAME)
+  /* Prepend the Process ID */
```

Comment:
```suggestion
  /* Prepend the Process name */
```
I tried to update my previous comment, but you were quicker on the force-push hehe.

Reply:
Np. Fixed as well.